### DISCUSSION TITLE
last_updated_on or last_update_on Discrepancies in ReadMe files and examples

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#71] [Q&A] [answered]
The ReadMe has the field last_updated_on, but there are examples that have last_update_on and some with last_updated_on.  Please clarify which should be used--- last_updated_on or last_update_on.

Here are some examples:
ReadMe has 

last_updated_on | Last Updated On | String | The date in which the file was last updated. Date must be in   an ISO 8601 format (e.g. YYYY-MM-DD)
-- | -- | -- | --

The In-network-rates JSON schema shows:  “update” without a ‘d’ on the end:
https://github.com/CMSgov/price-transparency-guide/blob/master/schemas/in-network-rates/in-network-rates.json 

row 97
"last_update_on": { "type": "string", "format": "date" },

Row 104:
"required": ["reporting_entity_name", "reporting_entity_type", "plan_name", "plan_id_type", "plan_id", "plan_market_type", "last_update_on"]

The in-network-rates XML schema shows:  “update” without a ‘d’ on the end:
https://github.com/CMSgov/price-transparency-guide/blob/master/schemas/in-network-rates/in-network-rates.xml

row 113 shows:
</last_update_on>

row 127 shows: 
<required>last_update_on</required>

But the fee for service examples for the in-network-rates files show “updated” with a ‘d’ on the end:

JSON Example:
https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-fee-for-service-sample.json 

Row 8
"last_updated_on": "2020-08-27",

XML Example:
https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-fee-for-service-sample.xml

Row 87:
  <last_updated_on>2020-08-27</last_updated_on>

**Comment by shaselton-usds** [comment, marked as answer]
Thanks again! Fix found here: https://github.com/CMSgov/price-transparency-guide/commit/0495fb434a933929345fb67881ce944f5c4d6ecc

If you feel comfortable, you are more than welcome to submit a PR if you find something like this.
